spec/support/factory_girl.rb

```diff
@@ -3,10 +3,15 @@
 FactoryGirl.define do
   factory :user do
     languages 'Navajo'
-    sequence(:username) {|n| "username#{n}" }
-    sequence(:email) {|n| "user#{n}@example.com" }
+    sequence(:username) { |n| "username#{n}" }
+    sequence(:email) { |n| "user#{n}@example.com" }
     password 'password'
     password_confirmation 'password'
   end
 
+  factory :bucket_list do
+    sequence(:title) { |n| "My List ##{n}" }
+    description 'I go places'
+    is_public 'false'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.